nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_listing-table-theme.scss

```diff
@@ -101,7 +107,7 @@
     }
 
     .mat-sort-header-arrow {
-        color: $canvas-primary-palette-900;
+        color: $surface;
```

Comment:
Here you define the sort arrow as `$surface` but [here](https://github.com/apache/nifi/pull/8480/files#diff-306fbee3e0c6d495e664b26016a798a91efce243b4b804cde92e76ef7eac654cR49) you define it as `$header-on-surface`. Which one is correct? I don't think we need both.